Formula/tensorflow_c.rb

```diff
@@ -0,0 +1,42 @@
+class TensorflowC < Formula
+  desc "C interface for Google's OS library for Machine Intelligence"
+  homepage "https://www.tensorflow.org/"
+  url "https://github.com/tensorflow/tensorflow", :using => :git,
```

Comment:
Please add `.git` at the end of the URL and remove `:using => :git` (it’s already the default).